index.md

```diff
@@ -1,7 +1,83 @@
-# Welcome to `nmstate`
+# We are nmstate!
+A descriptive network manager API for hosts.
 
-Hello hello hello.
+## What is it?
+A library with an accompanying cli tool that manages host networking settings
+in a descriptive manner. The networking state is described by a pre-defined
+schema, where reporting (current state) and changes (desired state) are both
```

Comment:
I'd break the sentence. ("where") is not so clear.

The networking state is described by a pre-defined schema. Reporting of current state and changes to it (desired state) both comf*o*m to it.